### DISCUSSION TITLE
Widget improvement

### DISCUSSION THREAD

**Discussion by rinka88** [templaza/astroid-framework#510] [Ideas]
![astroid307-new-widgets](https://github.com/templaza/astroid-framework/assets/40716604/cd2c519c-2260-462d-a3d0-aaea4d3fac6f)
Great idea about widgets out of the box. I see that for some jobs a builder is no longer needed.
Yoothteme pro has excellent functionality: Dynamic content that you can use joomla material with widgets. And none of the joomla templates have this.
https://www.youtube.com/watch?v=U1aomq1VLI0&t=67s&ab_channel=YOOtheme
If this happens here, it would be great.

**Comment by salahjaafar** [comment]
Good idea, and it will be another positive aspect for the Astroid framework. You can use this open-source solution to build pages: https://www.rsjoomla.com/joomla-extensions/free-joomla-page-builder.html
